Dedicated Hosting_Table.md

```diff
@@ -0,0 +1,72 @@
+# Dedicated Hosting_Table
```

Comment:
@asmita32675 I can't make a commit to your PR because its from a fork and i'm not a maintainer of the main repo.

So the first problem is this file and the other below have both been created in the root directory. After reading the docs it appears we need to move them into the content folder. However, it is using a static site generator called Hugo. Which means you have to run commands locally. I can help you if we can find a time to meet again. You can also see if punit can help you again since he helped before with the setup during your shift.

- [ ] We would want to clone your forked repo locally
- [ ] Checkout your branch by using command `git checkout asmita_docs`
- [ ] Look at the instructions again, specifically Step 3 in this section is important https://github.com/rackerlabs/support-how-to/blob/f19456c98e87b8ed60d28a392869969afdaed53d/CONTRIBUTING.md#create-an-article
- [ ] Instead of the sample line of code they provided I think we would want to run this line
`hugo new --kind support-bundle how-to/fabric/fabric-ticketing`
- [ ] Move the code from Fabric Ticketing Customer Facing Document.md to the new index.md inside the new fabric-ticketing folder the command created
- [ ] Run this `hugo new --kind support-bundle how-to/fabric/common-request-templates`
- [ ] Move the code from Dedicated Hosting_Table.md into the index.md file within the new common-request-templates folder
- [ ] If all code has been moved then delete the old files
- [ ] Use command `git add` followed by a file name to stage the commit
- [ ] Make a commit either with command or with vscode GUI button
- [ ] push back to your branch I think

Reply:
FYI at the bottom of this PR after every new commit you should be able to see a green check with the name `netlify/docs-support-how-to/deploy-preview — Deploy Preview ready!`. To the right of that you can click details to see a test deployment of what this PR looks like on the site.